src/main/java/org/openmrs/uitestframework/test/TestBase.java

```diff
@@ -339,6 +350,69 @@ public String patientIdFromUrl() {
     }
 
     /**
+     * Check if given patient exists
+     *
+     * @param patient The info for the patient to delete.
+     */
+    public void checkIfPatientExists(PatientInfo patient) throws DataSetException, SQLException, Exception {
+        if (patient.id != null) {
+            checkIfPatientExists(patient.id);
+        } else if (patient.uuid != null) {
+            deletePatientUuid(patient.uuid);
+        }
+    }
+
+    /**
+    * Check if patient with given id exists
+    * @param id
+     **/
+    public boolean checkIfPatientExists(String id) throws Exception{
```

Comment:
At a glance, the implementation of this method seems completely different from what the method name implies this should be...